File: CSharpCore/CSharpCore.Test2/HelloTest.cs
using Xunit;
using FluentAssertions;

namespace CSharpCore.Test
{
    public class HelloTest
    {
        [Fact]
        public void ShouldGreet()
        {
            Hello hello = new Hello();

            string greeting = hello.Greetings();

            greeting.Should().Be("Hello!");
        }
    }
}

File: Java/src/test/java/org/nagarro/HelloTest.java
package org.nagarro;



import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

public class HelloTest {

    @Test
    public void shouldGreet() {
        Hello hello = new Hello();

        String greeting = hello.greeting();

        assertThat("Hello World").isEqualTo(greeting);
    }

}
